### PR TITLE
Support sending apikey via headers in addition to sending it via query param

### PR DESCRIPTION
**Pull request by Iblis**
Implementation of #243

### CODE REVIEW

File: src/Service/CheckConsumerPermissionsService.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    /**
+     * @param Request $request
+     * @param Configuration $configuration
+     *
+     * @return void
+     */
+    public function performSecurityCheck(Request $request, Configuration $configuration): bool
+    {
+        $securityConfig = $configuration->getSecurityConfig();
+        if ($securityConfig['method'] === 'datahub_apikey') {
```
Comment (weisswurstkanone) [inline src/Service/CheckConsumerPermissionsService.php:32]: I think we should store this constant in a central place. wdyt ?

Comment (Iblis) [reply src/Service/CheckConsumerPermissionsService.php:32]: Good Idea. Do you have a class for Constants already? Or should we rather put it as a public const in Configuration.php, that is where it is used anyways, right?

**Comment by weisswurstkanone** [comment]
No, there is no didicated class for this but I guess Configuration.php should also be fine.
Your choice :-)

File: src/Configuration.php [hunk 2]
```diff
@@ -301,4 +301,6 @@ public function getSecurityConfig()
     {
         return $this->configuration['security'] ?? [];
     }
+
+    public const CONFIG_NAME_APIKEY = "datahub_apikey";
```
Comment (weisswurstkanone) [inline src/Configuration.php:305]: please move this up and prefix it with the "topic". So something like SECURITYCONFIG_AUTH_APIKEY

**Comment by weisswurstkanone** [comment]
thanks a lot!

**Pull request merged**